waku/node/waku_node.nim

```diff
@@ -114,14 +114,14 @@ proc getAutonatService*(rng: ref HmacDrbgContext): AutonatService =
   let autonatService = AutonatService.new(
     autonatClient = AutonatClient.new(),
     rng = rng,
-    scheduleInterval = some(chronos.seconds(120)),
+    scheduleInterval = Opt.some(chronos.seconds(120)),
     askNewConnectedPeers = false,
     numPeersToAsk = 3,
     maxQueueSize = 3,
     minConfidence = 0.7)
 
   proc statusAndConfidenceHandler(networkReachability: NetworkReachability,
-                                  confidence: Option[float]):
+                                  confidence: Opt[float]):
```

Comment:
Whats the difference between `Option[]` and `Opt[]` ?

edit: found my own answer :P
>You can use `Opt[T]` as a replacement for `Option` = `Opt` is an alias for
`Result[T, void]`, meaning you can use the full `Result` API on it: